cmd/models/onebuild-execution-plan.go

```diff
@@ -90,3 +99,27 @@ func longestPhaseAndCommandValue(executionPlan *OneBuildExecutionPlan) (string,
 func dashesOfLength(text string) string {
 	return strings.Repeat("-", len(text))
 }
+
+// PrintBanner prints the CommandContext's name in a 72-character banner
```

Comment:
`PrintBanner prints the CommandContext's name in a banner of the standard length`